implementations/rust/build.gradle

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
Let's add this line back :)

Reply:
Ah, my editor is super-zealous about white-space removal! I've put it back where I found it.